### PR TITLE
test(ui-breadcrumb): migrate old Breadcrumb tests

### PR DESCRIPTION
**Pull request by git-nandor**
Closes: INSTUI-3865

### CODE REVIEW

**Comment by balzss** [comment]
let's wait until #1308 is merged so the generated a11y tests can be added to this PR

**Review by joyenjoyer — APPROVED** [review]
Nice job with the ResizeObserver mock

**Review by joyenjoyer — APPROVED** [review]

**Review by balzss — APPROVED** [review]

**Pull request merged**
